.github/workflows/golang-fuzz-nightly.yml

```
@@ -29,10 +29,13 @@ jobs:
           GOARCH: ${{ matrix.goarch }}
         run: go test -failfast ./internal/par2 ./internal/bundle
 
-  fuzz-120min:
+  fuzz-60min:
     name: fuzz-${{ matrix.target.cat }}-${{ matrix.target.name }}-${{ matrix.goarch }}
     needs: unit-tests
     runs-on: ubuntu-latest
+    concurrency:
+      group: fuzz-nightly-${{ matrix.goarch }}
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)